src/dlgTriggerEditor.cpp

```diff
@@ -3660,15 +3660,15 @@ void dlgTriggerEditor::addScript(bool isFolder)
     saveScript();
     QString name;
     if (isFolder) {
-        name = "New Script Group";
+        name = tr("New script group");
     } else {
-        name = "NewScript";
+        name = tr("New script");
     }
     QStringList mainFun;
     mainFun << "-------------------------------------------------\n"
             << "--         Put your Lua functions here.        --\n"
             << "--                                             --\n"
-            << "-- Note that you can also use external Scripts --\n"
+            << "-- Note that you can also use external scripts --\n"
             << "-------------------------------------------------\n";
     QString script = mainFun.join("");
```

Comment:
Oooh! An erroneous use of a `""` instead of a `QString()` methinks!

Reply:
Again #2636